### ISSUE TITLE
cannot vendor pg_query_go

### ISSUE THREAD

**Issue by TotallyGamerJet** [pganalyze/pg_query_go#125]
This library cannot be vendored because `parser/include` doesn't contain any go files.

Error:

```
# github.com/pganalyze/pg_query_go/v6/parser
vendor/github.com/pganalyze/pg_query_go/v6/parser/parser.go:7:10: fatal error: 'pg_query.h' file not found
    7 | #include "pg_query.h"
      |          ^~~~~~~~~~~~
1 error generated.
```

Reproduce:

```go
package main

import _ "github.com/pganalyze/pg_query_go/v6"

func main() {}
```

```
go get github.com/pganalyze/pg_query_go/v6@latest
go mod vendor
go build
```

**Comment by lfittl** [comment]
@TotallyGamerJet Thanks for opening this issue! This is indeed a problem - I know that you can work around it using additional tools like [modvendor](https://github.com/goware/modvendor).

I'm curious, are you sure its just the missing `.go` files that are the problem, vs the fact that the files in question are not Go files at all? (if its the former, we may be able to do something, but my assumption has always been that Go's vendor tooling doesn't handle C language files)

**Comment by TotallyGamerJet** [comment]
Yes I am sure that it's the missing go files because the other C files in the parser package are present. It's just the ones in the sub directories which are only C and header files. We also came across this in Purego which was fixed by adding empty files.

https://github.com/ebitengine/purego/commit/aeed57cda74433391c1ebcd6c8ce5630cf944056
